packages/serverless-contracts/package.json

```diff
@@ -59,6 +58,7 @@
     "@middy/core": "^3.6.2",
     "@middy/error-logger": "^3.6.2",
     "@middy/http-cors": "^3.6.2",
+    "@swarmion/serverless-helpers": "^0.21.3",
```

Comment:
Are you sure about this? What do we use it for?

Reply:
it is used in the tests and benches